docs/USAGE-GUIDE.md

```diff
@@ -710,6 +710,61 @@ int s2n_shutdown(struct s2n_connection *conn,
 
 **s2n_shutdown** shuts down the s2n connection. Once a connection has been shut down it is not available for reading or writing.
 
+## Errors
+
+All s2n functions return 0 on success and -1 on failure. For failures, the thread local variable "s2n_errno" will be set with specific error information. Applications that simply wish to log the error can use a pattern like this:
```

Comment:
There is an error handling section that this could be added to (or remove the other section). There are some repetitive statements from both.